WalletWasabi.Gui/Global.cs

```diff
@@ -133,6 +133,20 @@ public Global(string dataDir, string torLogsFile, Config config, UiConfig uiConf
 		public async Task InitializeNoWalletAsync()
 		{
 			InitializationStarted = true;
+
+			#region ProcessKillSubscription
+
+			AppDomain.CurrentDomain.ProcessExit += (s, e) => DisposeAsync().ConfigureAwait(false).GetAwaiter().GetResult();
+
+			Console.CancelKeyPress += async (s, e) =>
+			{
+				e.Cancel = true;
+				Logger.LogWarning("Process was signaled for killing.", nameof(Global));
```

Comment:
```suggestion
				Logger.LogWarning("User asked to terminate the application.", nameof(Global));
```

CTRL+C is not killing the application. You can also override it so that the application won't actually stop. So I would suggest different wording for the message.

Reply:
ACK